crates/common/tedge_config/src/tedge_config.rs

```diff
@@ -566,7 +566,7 @@ impl ConfigSettingAccessor<RunPathDefaultSetting> for TEdgeConfig {
             .run
             .dir_path
             .clone()
-            .unwrap_or_else(|| self.config_defaults.default_logs_path.clone()))
+            .unwrap_or_else(|| self.config_defaults.default_run_path.clone()))
```

Comment:
Flock files should be created under `/run/lock`. Doesn't this return only `/run/`?

Reply:
P.S. This function should return just `/run`, this is what we ask for users. However, when creating a lock file, we have to postfix it to add `/lock`.